cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -54,6 +54,17 @@ describe('Top toolbar tests.', function() {
 			.should('exist');
 	});
 
+	it('Apply font size', function() {
+		cy.get('#tb_editbar_item_fontsizes')
+			.click();
+			
+		cy.contains('ul', '40')
+		   .click();
+
+		cy.get('#copy-paste-container p font')
+		    .should('have.attr', 'style', 'font-size: 40pt');
```

Comment:
You should not assert on '40pt' here, since the document contains this font size when the test opens it, so on this way we can't test whether the font size was changed.
Also cy.contains() method should select a different font size than '40'. That's why I suggested '60' here, so we can check whether the font size was actually changed.